src/main/webapp/js/index.js

```diff
@@ -127,3 +127,79 @@ $(function() {
           cb);
   cb(start, end);
 });
+
+
+/**
+ * Price range slider setup:
+ * https://refreshless.com/nouislider/.
+ */
+$(document).ready(function() {
+  const keypressSlider = document.querySelector('.slider-keypress');
+  const input0 = document.querySelector('.input-with-keypress-0');
+  const input1 = document.querySelector('.input-with-keypress-1');
+  const inputs = [input0, input1];
+
+  noUiSlider.create(
+      keypressSlider,
+      {start: [20, 80], connect: true, step: 1, range: {min: [0], max: [250]}});
+
+  keypressSlider.noUiSlider.on('update', function(values, handle) {
+    inputs[handle].value = values[handle];
+
+    /* begin Listen to keypress on the input */
+    function setSliderHandle(i, value) {
```

Comment:
Nested functions should be assigned a name using `const`.

https://google.github.io/styleguide/jsguide.html#features-functions-nested-functions